augur/data/__init__.py

```diff
@@ -0,0 +1,35 @@
+"""
+Resource files.
+"""
+# Copied without modification from nextstrain/cli/resources/__init__.py in the
+# Nextstrain CLI project¹.
+#
+# ¹ <https://github.com/nextstrain/cli/blob/a5dda9c0579ece7acbd8e2c32a4bbe95df7c0bce/nextstrain/cli/resources/__init__.py>
```

Comment:
@tsibley thanks for the pointer here. I like your approach in Nextstrain CLI and decided to copy it over here with some small modifications.

I've noticed in the past you've copied license files as well. Can you advise on the best approach here? Augur and Nextstrain CLI are under different licenses, so I'm thinking Nextstrain CLI's MIT license should be copied over.

Reply:
Yep, the license should be copied as well. Basically all open source licenses require that the license comes along with any copying.

Reply:
Copied the license over in cecd5e9063003ae95d6c014af9234f50396607d5.